### PR TITLE
support IPv6

### PR DESCRIPTION
**Pull request by sollhui**

### CODE REVIEW

File: spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/NacosDiscoveryProperties.java [hunk 1]
```diff
@@ -208,18 +215,15 @@ public class NacosDiscoveryProperties {
 	 */
 	private boolean ephemeral = true;
 
-	/**
-	 * Whether to enable nacos failure tolerance. If enabled, nacos will return cached
-	 * values when exceptions occur.
-	 */
-	private boolean failureToleranceEnabled;
```
Comment (steverao) [inline spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/NacosDiscoveryProperties.java]: Why did you delete the field?

Comment (sollhui) [reply spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/NacosDiscoveryProperties.java]: it's a mistake.I change it soon

File: spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/inetutil/UtilIPv6AutoConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+@ConditionalOnDiscoveryEnabled
+@ConditionalOnNacosDiscoveryEnabled
+public class UtilIPv6AutoConfiguration {
+
+	public UtilIPv6AutoConfiguration() {
+	}
+
+	@Bean
+	@ConditionalOnMissingBean
+	public InetIPv6Util inetipv6Util(InetUtilsProperties properties) {
```
Comment (steverao) [inline spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/inetutil/UtilIPv6AutoConfiguration.java]: Please modify the name from "inetipv6Util" to "inetIPv6Util"
